charts/kserve-resources/README.md

```diff
@@ -5,11 +5,11 @@
 First install the `kserve-crd` chart
 
 ```console
-helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version 0.11.2
+helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version v0.12.0-rc0
```

Comment:
This would change every time we release, does it default to the latest version ?